### DISCUSSION TITLE
How to center CTk.CheckBox?

### DISCUSSION THREAD

**Discussion by duruburak** [TomSchimansky/CustomTkinter#942] [Q&A] [answered]
`CTk.CheckBox` looks like this when there is no text, is there any way to center it?

![pic](https://user-images.githubusercontent.com/100048974/209454224-90efe2bd-cf0e-4e85-a3ec-689838d6b9de.png)

```
root = customtkinter.CTk()

frame = customtkinter.CTkFrame(root, border_width=3, border_color="red", fg_color="pink")
frame.pack(pady=10)

checkbox = customtkinter.CTkCheckBox(frame, border_color="yellow", bg_color="blue", text="")
checkbox.pack(pady=10, padx=10)

root.mainloop()
```

**Comment by Akascape** [comment, marked as answer]
@duruburak Add this line:
```python
checkbox._canvas.grid(column=1, padx=checkbox.cget("width")/2)
```
![Screenshot](https://user-images.githubusercontent.com/89206401/209468369-a9de6d05-9fad-4e36-8487-0b93b69b3deb.jpg)
